content/docs/engine-dev/tools/project-manager/source-code-map.md

```diff
@@ -0,0 +1,110 @@
+    - [ProjectManagerWindow](#projectmanagerwindow)  [[header](https://github.com/o3de/o3de/blob/development/Code/Tools/ProjectManager/Source/ProjectManagerWindow.h) ] [[cpp file](https://github.com/o3de/o3de/blob/development/Code/Tools/ProjectManager/Source/ProjectManagerWindow.cpp) ]
+        - Constructs the application's `QMainWindow` and `DownloadController`, and prepares all relevant transition screens for the Project Manager.
+    - [ScreensCtrl](#screensctrl)  [[header](https://github.com/o3de/o3de/blob/development/Code/Tools/ProjectManager/Source/ScreensCtrl.h) ] [[cpp file](https://github.com/o3de/o3de/blob/development/Code/Tools/ProjectManager/Source/ScreensCtrl.cpp) ]
+        - Container structure for all Project Manager Screens and GUI Widgets. Also houses primary transition code for facilitating Project Manager's business logic.
+    - ScreenFactory  [[header](https://github.com/o3de/o3de/blob/development/Code/Tools/ProjectManager/Source/ScreenFactory.h) ] [[cpp file](https://github.com/o3de/o3de/blob/development/Code/Tools/ProjectManager/Source/ScreenFactory.cpp) ]
+        - Helper class for ScreensCtrl that routes `ProjectManagerScreen` enums to appropriate ScreenWidget constructor, invokes that construtor, and returns an instance of a given screen for the Project Manager's business logic.
+
+
+### Application
+
+Inherits from `AzFramework::Application`.
+
+The functions of the Application class are exclusively run by `main.cpp`.
+
+The two primary functions to keep track of are `Init` and `Run`. [`TearDown`](https://github.com/o3de/o3de/blob/development/Code/Tools/ProjectManager/Source/Application.cpp#L260-L272) is invoked at `Application`'s deconstruction.
+
+
+##### [`Application::Init`](https://github.com/o3de/o3de/blob/development/Code/Tools/ProjectManager/Source/Application.cpp#L34)
+* Sets various `QApplication` and `QCoreApplication` attributes and parameters
+* Registers components for system logging, and invokes [`InitLog`](https://github.com/o3de/o3de/blob/development/Code/Tools/ProjectManager/Source/Application.cpp#L152)
+* Creates new instance of `QApplication`, feeding in command line arguments
+* Sets up PythonBindings
+* Calls [`RegisterEngine`](https://github.com/o3de/o3de/blob/development/Code/Tools/ProjectManager/Source/Application.cpp#L181) to setup the O3DE engine that Project Manager ships with
+* Gets connection to CommandLine, and parses for screen and projectPath parameters
+* Creates ProjectManagerWindow
+
+##### [`Application::Run`](https://github.com/o3de/o3de/blob/development/Code/Tools/ProjectManager/Source/Application.cpp#L274)
+* Sets up Qt GUI stylesheets
+* Sets up Window decoration and MainWindow Geometry
+* Shows the MainWindow
+* Run the QApplication's main loop
+
+[Back to Top](#map)
```

Comment:
I like the use of a "Back to Top" here. Update all of the links like this one to `#overview` if you accept my earlier header change suggestion.
```suggestion
[Back to Top](#overview)
```